assets/blocks/course-list-block/index.js

```diff
@@ -120,10 +122,83 @@ const hideUnnecessarySettingsForCourseList = () => {
 	} );
 };
 
+const getCourses = ( courseId ) => {
+	return useSelectWithDebounce(
+		( sel ) => {
+			const store = sel( coreDataStore );
+			const query = { id: courseId };
+			return {
+				courses:
+					store.getEntityRecords( 'postType', 'course', query ) || [],
```

Comment:
I'm not sure we need to import `store` and use it here. `select( 'core' ).getEntityRecords` should work. See https://github.com/Automattic/sensei/blob/trunk/assets/blocks/quiz/quiz-block/questions-modal/questions.js#L43 for a more thorough example.